app/models/manageiq/providers/google/event_catcher_mixin.rb

```diff
@@ -0,0 +1,26 @@
+module ManageIQ::Providers::Google::EventCatcherMixin
+  def parse_event_type(event)
+    event_type = event.fetch_path('structPayload', 'event_type')
+    event_subtype = event.fetch_path('structPayload', 'event_subtype')
+    if event_type.blank?
```

Comment:
for one statement lines, you would usually use the one line syntax
```ruby
event_type = 'unknown' if event_type.blank?
```

same for the remainder of this file